cli/src/commit_templater.rs

```diff
@@ -282,6 +296,16 @@ fn build_commit_method<'repo>(
                 commit.committer().clone()
             }))
         }
+        "signature" => {
+            template_parser::expect_no_arguments(function)?;
+            language.wrap_commit_signature(TemplateFunction::new(self_property, |commit| {
+                match commit.verification() {
+                    Ok(Some(v)) => CommitSignature::Present(v),
+                    Err(SignError::InvalidSignatureFormat) => CommitSignature::Invalid,
+                    _ => CommitSignature::Absent, // TODO: don't ignore verification errors
```

Comment:
fyi, I've added runtime error, so this function can return err instead of dummy `Absent` value.
https://github.com/martinvonz/jj/commit/0519954d4f63cf7d4a0aaf7641099caa0be879f9